### PR TITLE
Add readme and shared data example dockerfile

### PR DESCRIPTION
**Pull request by MrCreosote**
Dockerfile source is
https://github.com/StarRocks/demo/blob/master/documentation-samples/quickstart/docker-compose.yml

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        mc alias set myminio http://minio:9000 miniouser miniopassword
+        mc admin user svcacct add --access-key AAAAAAAAAAAAAAAAAAAA \
+        --secret-key BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB \
+        myminio \
+        miniouser
+    depends_on:
+      - minio
+
+  starrocks-fe:
+    image: starrocks/fe-ubuntu:3.2-latest
```
Comment (Tianhao-Gu) [inline docker-compose.yml:45]:
```suggestion
    image: starrocks/fe-ubuntu:3.2.8
```

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    hostname: starrocks-fe
+    container_name: starrocks-fe
+    user: root
+    command:
```
Comment (Tianhao-Gu) [inline docker-compose.yml:49]: we are going to use the command option in Rancher. This is only a temporary solution?

Comment (MrCreosote) [reply docker-compose.yml:49]: Nope, the next few PRs will be altering the repo to use env vars

File: docker-compose.yml (hunk 1 above)
Comment (MrCreosote) [reply docker-compose.yml:45]: This is the docker compose file as is from the starrocks git repo in rancher, I don't want to change it for this PR

Comment (Tianhao-Gu) [reply docker-compose.yml:45]: 👍

File: docker-compose.yml (hunk 2 above)
Comment (Tianhao-Gu) [reply docker-compose.yml:49]: 👍

**Review by Tianhao-Gu — APPROVED** [review]
👍

**Pull request merged**